include/mbedtls/mbedtls_config.h

```diff
@@ -3934,4 +3934,19 @@
  */
 //#define MBEDTLS_P256M_EXAMPLE_DRIVER_ENABLED
 
+/*
+ * Platform independent implementation for crypto algorithms.
```

Comment:
I think it is definitely wrong to have `Platform independent implementation for crypto algorithms.` as the first line of the description of `MBEDTLS_AES_USE_HARDWARE_ONLY`. The following line, `MBEDTLS_AES_USE_HARDWARE_ONLY` is the correct thing. This line is only confusing, and needs to be removed.